modules/programs/vscode.nix

```diff
@@ -28,6 +28,27 @@ in
         '';
       };
 
+      userKeybindings = mkOption {
+        type = types.listOf types.attrs;
```

Comment:
Would it be possible to have this with `types.attrsOf types.str`. I.e., the example becomes
```
{
  "ctrl+h" = "workbench.action.navigateLeft";
  "ctrl+l" = "workbench.action.navigateRight";
}
```

Reply:
It's possible to have the same keybindings with different `when`. For example (from the doc):
```json
{ "key": "f5",              "command": "workbench.action.debug.continue", "when": "inDebugMode" },
{ "key": "f5",              "command": "workbench.action.debug.start",    "when": "!inDebugMode" },
```